x-pack/plugins/security_solution/public/common/components/sessions_viewer/index.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import type { Filter } from '@kbn/es-query';
+import { SessionsComponentsProps } from './types';
+import { ESBoolQuery } from '../../../../common/typed_json';
+import { StatefulEventsViewer } from '../events_viewer';
+import { sessionsDefaultModel } from './default_headers';
+import { defaultRowRenderers } from '../../../timelines/components/timeline/body/renderers';
+import { DefaultCellRenderer } from '../../../timelines/components/timeline/cell_rendering/default_cell_renderer';
+import * as i18n from './translations';
+import { SourcererScopeName } from '../../store/sourcerer/model';
+import { getDefaultControlColumn } from '../../../timelines/components/timeline/body/control_columns';
+
+export const TEST_ID = 'security_solution:sessions_viewer:sessions_view';
+
+export const defaultSessionsFilter: Required<Pick<Filter, 'meta' | 'query'>> = {
+  query: {
+    bool: {
+      filter: [
+        {
+          bool: {
+            should: [
+              {
+                match: {
+                  'process.is_entry_leader': true,
```

Comment:
just about to test whether endpoint agent added this yet, but we will have a process.entry_leader.same_as_process: true prop which will act as a means to pull all entry leaders.

Reply:
here is tracking for this. https://github.com/elastic/security-team/issues/3470
we'll have to circle back and update this, for now probably can't wait for this fix.

Reply:
alright, I added a TODO to switch the fields once we have `same_as_process` working, also a todo to remove the `runtime_mapping` on the search strategy.

cc @kqualters-elastic